src/plugins/core/sheet.ts

```diff
@@ -766,7 +766,7 @@ export class SheetPlugin extends CorePlugin<SheetState> implements SheetState {
     this.history.update("sheetIdsMapName", sheetIdsMapName);
   }
 
-  private getDuplicateSheetName(sheetName: string) {
```

Comment:
mostly duplicated with helpers of migration, is it really useful to have both ? same for getNextSheetName